cdm/gcdm/build.gradle

```diff
@@ -0,0 +1,88 @@
+plugins {
+  // Provide convenience executables for testing
+  id 'application'
+  id 'com.github.psxpaul.execfork' version '0.2.2'
+}
+
+import com.github.psxpaul.task.JavaExecFork
+
+repositories {
+  gradlePluginPortal()
+  maven {
+    // For jj2000
+    url 'https://artifacts.unidata.ucar.edu/repository/unidata-releases/'
+  }
+}
+
+description = 'gRPC client and server implementation of CDM Remote Procedure Calls (gCDM).'
+ext.title = 'CDM Remote Procedure Calls'
+
+apply from: "$rootDir/gradle/any/java-library.gradle"
+apply from: "$rootDir/gradle/any/protobuf.gradle"
+apply from: "$rootDir/gradle/any/shared-mvn-coords.gradle"
+
+dependencies {
+  api project(':cdm:cdm-core')
+
+  implementation "io.grpc:grpc-protobuf:${depVersion.grpc}"
+  implementation "io.grpc:grpc-stub:${depVersion.grpc}"
+
+  compileOnly 'org.apache.tomcat:annotations-api:6.0.53'
+
+  runtimeOnly project(':bufr')
+  runtimeOnly project(':grib')
+  runtimeOnly "io.grpc:grpc-netty-shaded:${depVersion.grpc}"
+
+  testImplementation project(':netcdf4')
+
+  testImplementation enforcedPlatform(project(':netcdf-java-testing-platform'))
+  testImplementation project(':cdm-test-utils')
+  testImplementation 'com.google.truth:truth'
+  testImplementation 'junit:junit'
+  testImplementation 'commons-io:commons-io'
+  testImplementation 'org.mockito:mockito-core'
+  testImplementation "io.grpc:grpc-testing:${depVersion.grpc}"
+
+  // logging
+  implementation 'org.slf4j:slf4j-api'
+  runtimeOnly 'ch.qos.logback:logback-classic'
```

Comment:
do we want logback in the runtime or just the test runtime?

Reply:
We do want logback for the tests (I think this is what we usually use). But we also need a logger for `runtime` so that we can get the gcdm server logs (put in `$buildDir/gcdm_logs/gcdm.log`, see below). I don't know which implementation is preferred for this?

Reply:
`testRuntimeOnly` extends `runtimeOnly`, so we should be ok with only declaring the logging implementation as `runtimeOnly`. In terms of implementation, you can use whichever logging implementation you like (that's supported by slf4j), but the logging config files are logback specific, so you'd need to change those for whichever implementation you use (log4j2, log4j, etc.).

Reply:
Removed `testRuntimeOnly` in c86d0a118f. Left `runtimeOnly` logger implementation as logback-classic, let me know if you prefer something else!